neptune/alpha/internal/backends/hosted_neptune_backend.py

```diff
@@ -119,6 +119,21 @@ def create_experiment(self, project_uuid: uuid.UUID) -> Experiment:
         except HTTPNotFound:
             raise ProjectNotFound(project_id=project_uuid)
 
+    @with_api_exceptions_handler
+    def get_experiment_with_attributes(self, project_id: str, experiment_id: str) -> (Experiment, List[Attribute]):
+        params = {
+            'experimentId': project_id + '/' + experiment_id
```

Comment:
Are these names or uuids? We do not support experiment ids in form of `<UUID>/<UUID>`.